src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -6445,6 +6445,35 @@ GUI_TEST_CLASS_DEFINITION(test_6941) {
     CHECK_SET_ERR(!GTUtilsDashboard::hasNotifications(os), "Unexpected notification");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6952) {
+    //1. Click Tools -> BLAST -> Remote NCBI BLAST... Choose _common_data/fasta/human_T1_cutted.fa as input file
+    //2. CLick Next two times, then Run
+    //Expected state: no errors
+    class RemoteBLASTWizardFiller : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus &os) override {
+            QWidget *dialog = QApplication::activeModalWidget();
```

Comment:
nit: Use GTWidget::getActiveModelWidget(os) and you will not need the second line (checking for null)